oak-search-elastic/src/main/java/org/apache/jackrabbit/oak/plugins/index/elastic/query/ElasticRequestHandler.java

```diff
@@ -443,7 +445,7 @@ public PhraseSuggester suggestQuery() {
         return PhraseSuggester.of(ps -> ps
                 .field(FieldNames.SPELLCHECK)
                 .size(10)
-                .directGenerator(d -> d.field(FieldNames.SPELLCHECK).suggestMode(SuggestMode.Missing))
+                .directGenerator(d -> d.field(FieldNames.SPELLCHECK).suggestMode(SuggestMode.Missing).size(10))
```

Comment:
Could we document the "size(10)" part? I'm not sure if I understand... does it mean that we return a maximum of 10 results instead of 5? If yes why is 10 good and 5 bad, and would it make sense to document this or is it already documented (e.g. in the Lucene documentation)? Then we could point to that documentation.

But maybe I just completly misunderstand the "10" here.